### PR TITLE
Creating a test-shared module

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -56,6 +56,8 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     dynamicFeatures = [':about', ':designernews', ':dribbble']
+
```
Comment (keyboardsurfer) [inline app/build.gradle]: nit: remove these new lines

File: app/build.gradle [hunk 2]
```diff
@@ -66,8 +68,8 @@ repositories {
 
 dependencies {
     api project(':core')
-    api project(':bypass')
     implementation project(':search')
+
```
Comment (keyboardsurfer) [inline app/build.gradle]: Break between `api` and `implementation`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -49,5 +49,6 @@
     <string name="share">share</string>
     <string name="prospects_cant_post">Sorry but prospects cannot comment :(</string>
     <string name="bad_dribbble_shot_url">Can\'t open that link :(</string>
+    <string name="title_test_shared">Module Title</string>
```
Comment (keyboardsurfer) [inline app/src/main/res/values/strings.xml]: Not required for a non `dynamic-feature` module

File: test_shared/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="io.plaidapp.test.shared" >
+
```
Comment (keyboardsurfer) [inline test_shared/src/main/AndroidManifest.xml]: nit: duplicate line break

File: settings.gradle [hunk 5]
```diff
@@ -1,3 +1,3 @@
-include ':app', ':bypass'
+include ':app', ':bypass', ':test_shared'
```
Comment (keyboardsurfer) [inline settings.gradle]: move `:test_shared` to its own new include line

Comment (nickbutcher) [reply settings.gradle]: @keyboardsurfer why out of interest?

Comment (keyboardsurfer) [reply settings.gradle]: Some structure. These weren't added with any particular thought behind it afaict.

My preferred structure would be

* application
* dynamic-features
* first party libs
* third party libs

**Pull request merged**